src/ui/compilationStatusView.ts

```diff
@@ -9,6 +9,8 @@ const StatusBarPriority = 10;
 
 function toStatusMessage(status: CompilationStatus, message?: string | null): string {
   switch(status) {
+  case CompilationStatus.ResolutionStarted:
```

Comment:
Shouldn't this be when it starts parsing already?

Reply:
Yes, if resolution for the user is parsing + internal resolution, then resolution started is what we report.